src/main/java/data/Account.java

```diff
@@ -198,13 +198,13 @@ public String buildJsonConfig() {
   }
 
   /**
-   * Update existing account information, preventing changes to legacyAccountId
-   * and nextGenAccountId
+   * Update existing account information.
+   * Changes to legacyCustomerId, nextGenCustomerId, vendorId, and accountId
+   *      are disabled in the UI and not propagated to the backend.
+   *
    * @param request contains form data to edit existing account
    */
   public void updateExistingAccount(HttpServletRequest request) {
```

Comment:
please add a TODO here to prevent changes in the backend as well (even though we're not going to get to it, I think it's important to clarify it as a next step)